code/modules/events/bureaucratic_error.dm

```diff
@@ -0,0 +1,36 @@
+/datum/event/bureaucratic_error
+	announceWhen = 1
+	/// Jobs that are not allowed to be picked for the bureaucratic error
+	var/list/blacklisted_jobs = list(
+		/datum/job/assistant,
+		/datum/job/chief_engineer,
+		/datum/job/cmo,
+		/datum/job/rd,
+		/datum/job/hos,
+		/datum/job/ai,
+		/datum/job/captain,
+		/datum/job/hop,
+		/datum/job/nanotrasenrep,
+		/datum/job/blueshield,
+		/datum/job/judge,
+		/datum/job/chaplain
+	)
+
+/datum/event/bureaucratic_error/announce()
+	GLOB.major_announcement.Announce("A recent bureaucratic error in the Human Resources Department may result in personnel shortages in some departments and redundant staffing in others.", "Paperwork Mishap Alert")
+
+/datum/event/bureaucratic_error/start()
+	var/list/jobs = SSjobs.occupations.Copy()
+	var/datum/job/overflow
+	var/overflow_amount = rand(1, 6)
+	var/errors = 0
+	while(errors <= overflow_amount)
+		var/random_change = pick(-1, 1)
+		overflow = pick_n_take(jobs)
+		if(overflow.admin_only)
+			continue
+		if(overflow.type in blacklisted_jobs)
+			continue
+		overflow.total_positions = max(overflow.total_positions + random_change, 0)
+		log_and_message_admins("[overflow] slot changed by [random_change]")
```

Comment:
Can you instead add this to a list of strings. Then join those strings together into one log message? Having 6 different messages for a single event seems a bit overkill.

Reply:
Like this?